gnosis/eth/ethereum_network.py

```diff
@@ -803,6 +803,8 @@ class EthereumNetwork(Enum):
     RIKEZA_NETWORK_TESTNET = 12715
     QUANTUM_CHAIN_TESTNET = 12890
     SPS = 13000
+    CREDIT_SMARTCHAIN_MAINNET = 13308
```

Comment:
This would not be necessary. It is repeated in the next two lines.

Reply:
Removed